gapis/api/vulkan/externs.go

```diff
@@ -277,6 +277,46 @@ func (e externs) readMappedCoherentMemory(memory_handle VkDeviceMemory, offset_i
 		mem.Data.Slice(dstStart+r.Base, dstStart+r.Base+r.Size, l).Copy(
 			e.ctx, U8ᵖ(mem.MappedLocation).Slice(srcStart+r.Base, srcStart+r.Base+r.Size, l), e.cmd, e.s, e.b)
 	}
+
+	// if e.cmd.Extras() == nil || e.cmd.Extras().Observations() == nil {
```

Comment:
OOps, sorry, I forgot this piece of code. It was used to debugging the coherent memory observations...I will remove this.